straxen/corrections_services.py

```diff
@@ -105,16 +107,16 @@ def _get_correction(self, run_id, correction, global_version,
                         # on when something was processed therefore
                         # don't interpolate but forward fill.
                         df = self.interface.interpolate(df, when, how='fill')
+                    if correction in ('mlp_model', 'cnn_model', 'gcn_model'):
```

Comment:
I could also imagine that you want to add this to the `_get_corrections` function as an optional argument and then pass it from get_NN_file. This also works and is very clear.

Reply:
we could do that, lets see once we have more corrections, then we can have a better idea how we can improve the functionality of  `_get_corrections`